travis/pip.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# Used to return the result code given by pip instance of by grep
+# Used to return the result code given by pip instead of grep
```

Comment:
The original comment is weird but does the suggested change really make sense? :P